### PR TITLE
Update docs + restructure files

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Update docs + restructure files

In this first PR, a main README is added and updated, metadata for the first plate is added and a platemap figure is generate, and a notebook is added to restructure the files into one folder.

### CODE REVIEW

File: 0.download_data/metadata/platemap_figures/NF0014_platemap_figure.png
Comment (MattsonCam) [inline 0.download_data/metadata/platemap_figures/NF0014_platemap_figure.png:1]: The platemap looks nice

Comment (MattsonCam) [reply 0.download_data/metadata/platemap_figures/NF0014_platemap_figure.png:1]: I'm having a hard time telling some of the colors apart. Could also change the shape of the well circles to reduce the number of colors needed. Another idea: add the first three letters of each treatment inside each well circle. The last idea: include multiple platemap figures with a fraction of the total treatment, e.g. only color half of the treatments for png1 and grey out the rest, and then on another figure (call it png2) add the other half of the treatments and grey out the treatments from png1, so essentially 2 figures.

File: 0.download_data/scripts/update_file_structure.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            if second_level_folder.is_dir():
+                # Go into the second folder
+                for third_level_folder in second_level_folder.iterdir():
+                    if third_level_folder.is_dir():
+                        # Get all images inside the third-level folder
+                        for image_file in third_level_folder.iterdir():
+                            if image_file.suffix.lower() in image_extensions:
+                                # Copy each image to the NF0014 folder
+                                destination = nf0014_dir / image_file.name
+                                shutil.copy2(image_file, destination)  # copy2 preserves metadata
```
Comment (MattsonCam) [inline 0.download_data/scripts/update_file_structure.py:55]:
```suggestion
for image_file in parent_dir.rglob('*/*/*'):
    if image_file.is_file() and image_file.suffix.lower() in image_extensions:
        destination = nf0014_dir / image_file.name
        shutil.copy2(image_file, destination)
```
May consider refactoring similar to suggestion above

File: 0.download_data/scripts/update_file_structure.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+print("All images have been copied to the NF0014 folder!")
```
Comment (MattsonCam) [inline 0.download_data/scripts/update_file_structure.py:57]: May consider writing code to error check this

File: 0.download_data/update_and_visualize.sh
Comment (MattsonCam) [inline 0.download_data/update_and_visualize.sh:1]: Could also use `just` like Dave was using, or something similar, but this works too

File: README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# GFF NF1 organoid image profiling
+
+In this repository, we perform image analysis and image-based profiling on NF1 organoids derived from patients that have been treated with a modified Cell Painting assay.
```
Comment (MattsonCam) [inline README.md:3]:
```suggestion
In this repository, we perform image analysis and image-based profiling on NF1 organoids derived from patients treated with a modified Cell Painting assay.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# GFF NF1 organoid image profiling
```
Comment (MattsonCam) [inline README.md:1]: Maybe this comes later in the future, but could describe NF1 more or provide links, and other background info about the organoid experiment data.

**Review by MattsonCam — APPROVED** [review]
Just left a few comments, otherwise, LGTM @jenna-tomkinson. Good job!

File: 0.download_data/metadata/platemap_figures/NF0014_platemap_figure.png
Comment (jenna-tomkinson) [reply 0.download_data/metadata/platemap_figures/NF0014_platemap_figure.png:1]: Thanks! I decided to find a better palette than the default from ggplot2. I think those are very good suggestions, but I think it is more concise to keep this one figure for now. The new palette I have added is color-blind friendly and each color is clearly distinct from each other. I appreciate you pointing it out!

File: 0.download_data/scripts/update_file_structure.py (hunk 1 above)
Comment (jenna-tomkinson) [reply 0.download_data/scripts/update_file_structure.py:55]: This is a very cool and simplified version! I have added this change but had to make an edit to include one more `*`: `for image_file in parent_dir.rglob('*/*/*/*')` as it wasn't going into the right directory to find images. Now it is good, thanks!

File: 0.download_data/update_and_visualize.sh
Comment (jenna-tomkinson) [reply 0.download_data/update_and_visualize.sh:1]: Agreed! I think `just` is out of scope for this first PR, I think we can include down the line once we likely have a bunch more environments to perform different pipelines.

File: README.md (hunk 3 above)
Comment (jenna-tomkinson) [reply README.md:3]: Updated!

File: README.md (hunk 4 above)
Comment (jenna-tomkinson) [reply README.md:1]: I think most of this comes later. NF1 links could be helpful and the only other experiment info that could be included is the microscope which I don't remember off the top of my head.

I think in further PRs this README will be updated more to include things as we progress.

**Comment by jenna-tomkinson** [comment]
@MattsonCam Thank you for the review! I am merging now!

**Pull request merged**